graphql.go

```diff
@@ -34,7 +34,7 @@ func ParseSchema(schemaString string, resolver interface{}, opts ...SchemaOpt) (
 		opt(s)
 	}
 
-	if err := s.schema.Parse(schemaString); err != nil {
+	if err := s.schema.Parse(schemaString, s.noCommentsAsDescriptions); err != nil {
```

Comment:
Should we name the variable `useStringComments` instead? I think that shorter is better. WDYT?

Reply:
I'll rename it to useStringsAsDescriptions. The spec uses the term "description" so I prefer to use that here too.

Reply:
Actually, `useStringDescriptions` is as short as I can make it.